src/IBM.Watson.TextToSpeech.v1/Examples/ServiceExample.cs

```diff
@@ -336,7 +336,7 @@ public void DeleteUserData()
                 );
 
             Console.WriteLine(result.StatusCode);
-        }
+        }
```

Comment:
Any reason why this file has no real changes?

Reply:
We moved the websocket example from here to the StreamingExample folder

Reply:
Most likely hidden changes in line endings